app/Controllers/RegistrationController.php

```
@@ -260,12 +260,30 @@ public function verifyEmail(Request $request, Response $response, mysqli $db): R
         if ($row = $res->fetch_assoc()) {
             $uid = (int) $row['id'];
             $stmt->close();
-            $stmt = $db->prepare("UPDATE utenti SET email_verificata = 1, token_verifica_email = NULL, data_token_verifica = NULL WHERE id = ?");
+
+            // Whether new self-registrations still need an administrator to
+            // approve them AFTER they verify their email. Default true (the
+            // historical behaviour). When false, verifying the email is enough
+            // to activate the account.
+            $requireApproval = (bool) ConfigStore::get('registration.require_admin_approval', true);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ConfigStore' in method 'verifyEmail'. (undefined)

(StaticAccess)